### ISSUE TITLE
feature reqjes

### ISSUE THREAD

**Issue by jessicajonesunitedkingdom** [emoon/HippoPlayer#105]

**Comment by jessicajonesunitedkingdom** [comment]
It could be usefull to integrate a ftp client for downloading music files from servers

here is an android app doing this for adlib music ftps. 

https://play.google.com/store/apps/details?id=com.omicronapplications.andplug&hl=nl&referrer=utm_source%3Dgoogle%26utm_medium%3Dorganic%26utm_term%3Dandplug+android

I use this and it's very easy and fast to get files from servers.
You can choose to play immediately after download or not.


here is a rust ftp client

https://github.com/mattnenterprise/rust-ftp

**Comment by emoon** [comment]
Yes. This is something I have planned on doing, but haven't got around to yet.
